api/v1alpha4/machinedeployment_types.go

```diff
@@ -57,6 +60,11 @@ type MachineDeploymentSpec struct {
 	// +kubebuilder:validation:MinLength=1
 	ClusterName string `json:"clusterName"`
 
+	// RolloutAfter performs a rollout of the MachineDeployment
+	// after a given time.
+	// +optional
+	RolloutAfter *metav1.Time `json:"rolloutAfter,omitempty"`
```

Comment:
Thinking about how the end user will interpret this description, I'm not sure from this what this actually means. 

Does it mean that it delays the next rollout until this time, or does it mean it will just rollout a fresh set of machines after this period? Also, what is expected if this is in the past, should I as a user clear the field? Will something clear it for me? 

Is this an appropriate place to improve the documentation? It would end up in kubectl describe if we did right?

Reply:
Updated the comment. See if that helps.